### PR TITLE
Extension name format not showing

### PR DESCRIPTION
**Pull request by rajesh-mage**
## Purpose of this pull request

This pull request (PR) Extension name format not showing on the Frontend view.

extension-format-not-showing

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/extension-dev-guide/build/enable-module.html

### CODE REVIEW

**Review by BarnyShergold — APPROVED** [review]
Good fix but you seem to have a conflict on the PR

**Comment by dobooth** [comment]
@rajesh-mage Yeah, 568 changed files in this PR. Your branch needs updating for sure.

**Review by dshevtsov — CHANGES_REQUESTED** [review]
Make sure you created your branch from the `master` branch. You can try to rebase your branch onto the latest `master` branch, and force push.

**Comment by rajesh-mage** [comment]
Hi @dshevtsov @BarnyShergold @dobooth 
Conflict issue resolved.

Thanks.

**Review by BarnyShergold — APPROVED** [review]

File: src/guides/v2.3/extension-dev-guide/build/enable-module.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ After you have built the component and are ready to enable it in your Magento en
    bin/magento module:status <extension-name>
    ```
 
-   An extension name uses the format: <VendorName>_<ComponentName>.
+   An extension name uses the format: VendorName_ComponentName.
```
[the remark this answers is not in the document]
Comment (dshevtsov) [reply src/guides/v2.3/extension-dev-guide/build/enable-module.md:31]: We typically use less-than and greater-than signs to indicate placeholders. To make them appear, they should be formatted as code or escaped.

**Review by dshevtsov — CHANGES_REQUESTED** [review]

File: src/guides/v2.3/extension-dev-guide/build/enable-module.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (rajesh-mage) [reply src/guides/v2.3/extension-dev-guide/build/enable-module.md]: Hi @dshevtsov,
Thanks for your review. The requested change is done.

Thanks.

**Review by ajithkumar-maragathavel — APPROVED** [review]
Approved. Looks good for me.

**Review by dshevtsov — APPROVED** [review]

**Comment by dshevtsov** [comment]
@magento import code to magento-devdocs/devdocs

**Comment by magento-engcom-team** [comment]
@dshevtsov the branch with code successfully imported into`magento-devdocs/devdocs` repository. Branch name: `imported-magento-devdocs-8936`.

**Pull request merged**